pkg/flowkit/output/prompt.go

```diff
@@ -507,3 +505,57 @@ func ReportCrash() bool {
 
 	return chosen == 0
 }
+
+func CreateAccountNetworkPrompt() (string, config.Network) {
+	networkNames := make([]string, len(config.DefaultNetworks()))
+	for i, net := range config.DefaultNetworks() {
+		networkNames[i] = net.Name
+	}
+
+	networkPrompt := promptui.Select{
+		Label: "Which network do you want to create the account on?",
+		Items: []string{"Local Emulator", "Flow Testnet", "Flow Mainnet"},
```

Comment:
I'ma bit nervous about hardcoding these values and assuming the indexes are right. Maybe add a test to make sure they don't get out of order.

Reply:
I think we should just use dictionary to address the index problem

Reply:
addressed